setup.sh

```diff
@@ -105,7 +110,7 @@ if [ -d "$LEAPP" ]; then
 
     # If we found at least one available session, then we can presume
     # this installation was successful.
-    if (( $(echo "$AVAILABLE_LEAPP_SESSIONS" | wc -l) > 10 )); then
+    if (( $(echo "$AVAILABLE_LEAPP_SESSIONS" | wc -l) > 0 )); then
```

Comment:
This was a typo while testing.  We're looking for at least 1 session to be loaded to consider this a success.